src/entropic/sinkhorn.jl

```diff
@@ -504,3 +504,85 @@ function sinkhorn_barycenter(μ, C, ε, w; tol=1e-9, check_marginal_step=10, max
     end
     return u[:, 1] .* (K * v[:, 1])
 end
+
+"""
+    sinkhorn_divergence(
+        c,
+        μ::Union{FiniteDiscreteMeasure, DiscreteNonParametric},
+        ν::Union{FiniteDiscreteMeasure, DiscreteNonParametric},
+        ε; regularization=false, plan=nothing, kwargs...
```

Comment:
One should not include the regularization keyword argument since it is not supported.

```suggestion
        ε; plan=nothing, kwargs...
```